content/package-builder/pkg-deps.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "How are dependencies installed?"
+date: 2018-01-28T21:55:52+01:00
+anchor: "pkg-deps"
+weight: 18
+
+---
+
+The R-hub package builder installs all package dependencies on-demand, at build/check time, from CRAN,
+and possibly other sources, using the [`remotes` package](https://remotes.r-lib.org/). 
+
+If your package depends on a package that is on CRAN that was only very recently released and that does not have a binary build yet, R-hub will try to build it. If it requires compilation,
```

Comment:
Maybe start with "On platforms that make use of binaries, typically Windows and macOS,"